### DISCUSSION TITLE
Is there any alternative cross-platform code on this one since System.Drawing.Common only supported on Windows?

### DISCUSSION THREAD

**Discussion by GeraldEugenio** [baSSiLL/SharpAvi#50] [Q&A]
https://github.com/baSSiLL/SharpAvi/blob/02b1447c5a79db9cf0ee12cbbb3bbba3dc86cb82/Sample/Recorder.cs#L222

**Comment by baSSiLL** [comment]
Getting a screenshot is an OS-specific operation, so any cross-platform solution would do it via the native interop on each platform.
Check cross-platform UI frameworks for .NET (like Avalonia, MAUI), some of them probably contain this function.
If you're actually interested in a snapshot of a window of your application, then most UI frameworks should support this.
